writehat/templates/widgets/custom_date.html

```diff
@@ -0,0 +1,12 @@
+{% include 'django/forms/widgets/multiwidget.html' %} <a href="#id_reportDate_day" id="set_today_val">Today</span>
```

Comment:
are there plans to add styles to make this look more like a button rather than a hyperlink?

Reply:
I can; does it look out of place as is?